crates/sui-types/src/sui_system_state.rs

```diff
@@ -277,6 +283,63 @@ impl SuiSystemState {
         }
     }
 
+    pub fn get_current_epoch_peer_ids(&self) -> HashMap<PeerId, String> {
```

Comment:
Seems like this method should return `HashMap<PeerId, SomeEnum>`, rather then a String with pre-defined values like `"other_primary"`, `"other_worker"`, etc.